server/mediaserver.go

```diff
@@ -425,10 +480,15 @@ func gotRTMPStreamHandler(s *LivepeerServer) func(url *url.URL, rtmpStrm stream.
 			return ErrRTMPPublish
 		}
 		LastManifestID = mid
-		vParams := ffmpeg.VideoProfileToVariantParams(ffmpeg.VideoProfileLookup[vProfile.Name])
-		manifest.Append(fmt.Sprintf("%v.m3u8", hlsStrmID), pl, vParams)
 
-		s.LivepeerNode.VideoSource.UpdateHLSMasterPlaylist(mid, manifest)
+		var jid int64
+		if jobId != nil {
+			jid = jobId.Int64()
+		}
+		for _, os := range drivers.Storages {
+			session := os.StartSession(jid, string(mid), nonce)
```

Comment:
I would call this something like `storageSession`.  Otherwise someone can confuse it as a video session when it appears in the code later.